geode-core/src/main/java/org/apache/geode/distributed/internal/locks/DLockToken.java

```diff
@@ -87,7 +87,8 @@
   private Thread thread;
 
   /**
-   * Number of threads currently using this lock token.
+   * Number of usages of this lock token. usageCount = recursion + (# of threads waiting for this
+   * lock). It's weird, I know.
```

Comment:
I could go either way. I don't mind a bit of humour and dialogue in our codebase.